src/data-base/GameData.js

```diff
@@ -0,0 +1,24 @@
+const GameModel = require("../model/GameModel");
+const InitialConnect = require("./SequelizeConnection");
+const ConfigConnect = require("../config/ConfigConnect.js");
```

Comment:
should be ConnectionConfig

Reply:
The change was made according to the suggestion for name the file DatabaseConnection